src/request/sign-transaction/SignTransactionApi.js

```
@@ -43,9 +42,10 @@ class SignTransactionApi extends TopLevelApi {
      */
     parseLayout(layout) {
         if (!layout) {
-            return SignTransactionApi.Layouts.standard;
+            return SignTransactionApi.Layouts.STANDARD;
         }
-        if (!SignTransactionApi.Layouts[layout]) {
+        // @ts-ignore
```

You removed the ignored error reference.